site/src/main/paradox/FAQ.md

```diff
@@ -94,9 +94,10 @@ object MyJob {
   def main(cmdlineArgs: Array[String]): Unit = {
     val (sc, args) = ContextAndArgs(cmdlineArgs)
 
-    val collections =
-      Seq("gs://bucket1/data/*.avro", "gs://bucket2/data/*.avro")
-        .map(sc.avroFile[TestRecord](_))
+    val collections = Seq(
+      "gs://bucket1/data/",
+      "gs://bucket2/data/"
+    ).map(path => sc.avroFile[TestRecord](path, suffix=".avro"))
```

Comment:
Is this the way we want to recommend users perform reads going forward? I marginally prefer the old version since it _seems_ more compact & explicit, but that might just be Stockholm syndrome